app/models/ingest_job.rb

```diff
@@ -650,15 +650,17 @@ def delete_auto_differential_expression_results
 
   # read the DE manifest file generated during ingest_differential_expression to create DifferentialExpressionResult
   # entry for given annotation/cluster, and populate any one-vs-rest or pairwise_comparisons
-  def create_user_differential_expression_results
+  def create_author_differential_expression_results
     de_info = study_file.differential_expression_file_info
     cluster_group = de_info.cluster_group
     annotation_identifier = "#{de_info.annotation_name}--group--#{de_info.annotation_scope}"
     Rails.logger.info "Creating differential expression result object for annotation: #{annotation_identifier} from " \
                       "user-uploaded file #{study_file.upload_file_name}"
     de_result = DifferentialExpressionResult.new(
       study:, study_file:, cluster_group:, cluster_name: cluster_group.name, is_author_de: true,
-      annotation_name: de_info.annotation_name, annotation_scope: de_info.annotation_scope
+      annotation_name: de_info.annotation_name, annotation_scope: de_info.annotation_scope,
```

Comment:
```suggestion
      annotation_name: de_info.annotation_name, annotation_scope: de_info.annotation_scope, computational_method: de_info.computational_method,
```

Blocker:  We're not setting `computational_method` on the newly created `DifferentialExpressionResult` document.  This will cause the default of `wilcoxon` to be set, but in my testing I passed `logreg`.  This breaks URLs to output files in the bucket.  Ingest did the right thing, and I see files in the bucket with `logreg`, and the `DifferentialExpressionFileInfo` embedded document also says `logreg`.  It's just not getting passed here.  I think this was actually an existing issue, but since we're touching this method we should fix this now.